api/main_endpoints/routes/User.js

```diff
@@ -509,4 +509,36 @@ router.post('/usersSubscribedAndVerified', function(req, res) {
     });
 });
 
+// Search for all members with verified emails, subscribed, and not banned or pending
+router.post('/usersValidVerifiedAndSubscribed', function(req, res) {
+  if (!checkIfTokenSent(req)) {
+    return res.sendStatus(FORBIDDEN);
+  } else if (!checkIfTokenValid(req, membershipState.OFFICER)) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  User.find({
+    emailVerified: true,
+    emailOptIn: true,
+    accessLevel: { $gte: membershipState.NON_MEMBER }
+  })
+    .then((users) => {
+      const data = users.map(user => [user.email]);
+      data.unshift(['email']);
+
+      res.statusCode = 200;
+      res.setHeader('Content-Type', 'text/csv');
+      data.forEach(function(item) {
+        res.write(item.map(function(field) {
+          return '"' + field.toString().replace(/\"/g, '""') + '"';
+        }).toString() + '\r\n');
+      });
```

Comment:
for the other comment about `users.map`, can we then do:
```suggestion
        res.write(item.email+ '\r\n');
      });
```
?